src/frequenz/sdk/timeseries/_resampling.py

```diff
@@ -744,14 +744,43 @@ def resample(self, timestamp: datetime) -> Sample[Quantity]:
         # resort to some C (or similar) implementation.
         relevant_samples = list(itertools.islice(self._buffer, min_index, max_index))
         if not relevant_samples:
-            _logger.warning("No relevant samples found for: %s", self._name)
+            self._log_no_relevant_samples(minimum_relevant_timestamp, timestamp)
+
         value = (
             conf.resampling_function(relevant_samples, conf, props)
             if relevant_samples
             else None
         )
         return Sample(timestamp, None if value is None else Quantity(value))
 
+    def _log_no_relevant_samples(
+        self, minimum_relevant_timestamp: datetime, timestamp: datetime
+    ) -> None:
+        """Log that no relevant samples were found.
+
+        Args:
+            minimum_relevant_timestamp: Minimum timestamp that was requested
+            timestamp: Timestamp that was requested
+        """
+        if not _logger.isEnabledFor(logging.DEBUG):
```

Comment:
This should be `WARNING`!
```suggestion
        if not _logger.isEnabledFor(logging.WARNING):
```

Reply:
Shit. So much for my FCR deployment.